cypress/e2e/shared/tasks.test.ts

```diff
@@ -132,13 +132,18 @@ describe('Tasks - TSM', () => {
 
     cy.focused()
 
-    cy.getByTestID('flux-editor').monacoType(`option task = {
+    cy.getByTestID('flux-editor')
+      .monacoType(
+        `option task = {
     name: "Option Test",
     every: 24h,
     offset: 20m
   }
   from(bucket: "defbuck")
-    |> range(start: -2m)`)
+    |> range(start: -2m)`
+      )
+      // Fix bug with monaco error inserting an extra } here.
+      .monacoType('{backSpace}')
 
```

Comment:
This test wasn't working correctly - the monaco editor was adding an extra `}` at the end, and the old regex was deleting that too - likely caught by the greedy quantifier `*`. The new regex has a lazy quantifier (`*?`), so it will (correctly) only match the first closing bracket from `option task`.

Reply:
Unfortunately, since this is a monaco-editor issue, the easily work around is just to delete the `}` it's improperly adding at the end.

Reply:
Nice!